cscs-checks/apps/amber/amber_check.py

```diff
@@ -18,8 +18,14 @@ def __init__(self, name, input_file, output_file, **kwargs):
             ('-O -i %s -o %s' % (input_file, output_file)).split()
         self.keep_files = [output_file]
 
-        self.sanity_patterns = sn.assert_found(
-            r'Final Performance Info:', output_file)
+        energy = sn.extractsingle(r' Etot\s+=\s+(?P<energy>\S+)',
+                                  output_file, 'energy', float, item=-2)
+        energy_reference = -443246.8
+        energy_diff = sn.abs(energy-energy_reference)
```

Comment:
Please leave whitespace around the `-` operator. Check out the [coding style](https://github.com/eth-cscs/reframe/wiki/Coding-Style-Guide#formatting-operators).